app/controllers/post_controller.rb

```diff
@@ -0,0 +1,33 @@
+class PostController < ApplicationController
+  before_action :signedin_user, only: [ :new, :create]
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(post_params)
```

Comment:
There is two way to do this. The convenient way is `current_user.posts.build(post_params)`. Another way is 
```ruby
@post=Post.new(post_params)
@post.user_id = current_user.id
```
Now, it's your choice which one you want to choose.

Reply:
chose the build option